src/main/python/hydra/rmqtest/rmq_sub.py

```diff
@@ -58,7 +58,11 @@ def run10(argv):
     pwd = os.getcwd()
     l.info("CWD = " + pformat(pwd))
     cmgr = ChildManager()
+    myenv = os.environ.copy()
     cmd = './hydra hydra.rmqtest.rmq_sub.run'.split(' ') + argv[1:]
+    # TODO: (AbdullahS) Find a better way to do this
+    if "mock" in myenv:
```

Comment:
./ is run from the target/src/.. directory on remote slaves in case of mesos/marathon, the same path is not true for local test so the command has to be modified, hydra is in the path, so just hydra works.